### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by bf4**
CI output needs to be reviewed.

I'm also trying to figure out which tests cause
- `lib/action_dispatch/routing/route_set.rb:429: warning: instance variable @_routes not initialized`
and why.
- If we can silence all the grape warnings `gems/grape-0.15.0/lib/grape`

Also:
- Less hacky test routes setup
- Remove annoying progress reporter broken by warnings output

### CODE REVIEW

**Comment by groyoh** [comment]
Looks like the Rails routes are broken.

File: test/support/rails_app.rb [hunk 1]
```diff
@@ -16,12 +16,20 @@ class ActiveModelSerializers::RailsApplication < Rails::Application
 end
 ActiveModelSerializers::RailsApplication.initialize!
 
+Rails.application.routes.draw do
+  get ':controller(/:action(/:id))'
+  get ':controller(/:action)'
+end
 module TestHelper
-  Routes = ActionDispatch::Routing::RouteSet.new
```
Comment (groyoh) [inline test/support/rails_app.rb]: Tried it out locally, seems like removing this broke the routes.

**Comment by groyoh** [comment]
The routes and grape warning are still quite annoying. Looking forward to get rid of them. Good job!

**Comment by bf4** [comment]
I have a fix for routes

**Pull request merged**